ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -254,16 +257,25 @@ export default React.memo<Props>(function VideoJs(props: Props) {
       // set playsinline for mobile
       player.children_[0].setAttribute('playsinline', '');
 
+      // immediately show control bar while video is loading
+      // $FlowFixMe
+      document.querySelector('.vjs-control-bar').style.setProperty('opacity', '1', 'important');
+      // $FlowFixMe
+      document.querySelector('.vjs-control-bar').style.setProperty('display', 'flex');
```

Comment:
The standard methods for React and videojs framework:
- use the `player` API, like `player.controls(true)`, `addClass`, etc.
- css specificity
    - `https://github.com/videojs/video.js/issues/5143`
        - I didn't link directly to avoid creating an unnecessary mention in their repo.
    - https://blog.videojs.com/hiding-and-showing-video-player-controls/